### ISSUE TITLE
Include the browselists documentation in Orangelight

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#2845]
Browselists are specific to orangelight include the documentation in this repo as well. https://github.com/pulibrary/bibdata/blob/main/docs/browse-list.md

**Comment by maxkadel** [comment]
Would it be better to have in only one repo and link from the other? Otherwise they'll get out of sync and it won't be clear which is the most up to date. If it's only in one, does it make more sense for it to be in orangelight?

**Comment by christinach** [comment]
@maxkadel yes it makes sense to have the browselists documentation in one repo. And it makes more sense to be in Orangelight.
